### ISSUE TITLE
Could not initialize class com.android.build.gradle.internal.VariantManager

### ISSUE THREAD

**Issue by SantoshKumar4547** [facebook/react-native#24846]
info JS server already running.
info Building and installing the app on the device (cd android && ./gradlew app:installDebug)...

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/santoshkumar/sites/AwesomeProject/android/app/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':app'.
> Could not initialize class com.android.build.gradle.internal.VariantManager

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

React Native version:

info 
  React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.6
      CPU: (4) x64 Intel(R) Core(TM) i5-5250U CPU @ 1.60GHz
      Memory: 53.02 MB / 8.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.3 - /usr/local/bin/node
      npm: 6.4.1 - /usr/local/bin/npm
    SDKs:
      Android SDK:
        API Levels: 28
        Build Tools: 28.0.3
        System Images: android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.4 AI-183.5429.30.34.5452501
      Xcode: /undefined - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.8 => 0.59.8 
    npmGlobalPackages:
      react-native-cli: 2.0.1
## Steps To Reproduce

1. while running command react-native run-android 
 the system throws the above ERROR

**Comment by shreyasnisal** [comment]
Could someone please help me with this issue. I am facing this and nothing has helped till now. I have tried different versions of node, npm. And also reinstalling these as well as the react-native-cli. My projects which I was able to build successfully earlier are not running either, nor are fresh projects.

**Comment by NumOne01** [comment]
hey could you solve it ?

**Comment by shreyasnisal** [comment]
No. Turns out that I had to make a lot of changes to my OS, and when I ran it again, it was working fine. I don't think this would solve the error, but are you using a windows machine with spaces in your user account name?
